src/tests/class_tests/openms/data/ExperimentalDesign_input_1.tsv

```diff
@@ -0,0 +1,29 @@
+Fraction_Group	Fraction	Spectra_Filepath	Label	Sample
+1	1	JD_06232014_sample1-A.raw	1	1
+2	1	JD_06232014_sample1_B.raw	1	2
+3	1	JD_06232014_sample1_C.raw	1	3
```

Comment:
@timosachsenberg can you please make an example file that covers multiple cases, e.g. multiple fractionations or a fractionation combined with TMT ? so that people have some idea what such a file would look like

Reply:
done in ExperimentalDesign_input_2.tsv fractionated 4plex